src/utils/sort.js

```diff
@@ -0,0 +1,24 @@
+export function sortDeviceInfos(deviceInfos, sortTokens) {
+  if (sortTokens[0] === "name") {
+    if (sortTokens[1] === "desc") {
+      deviceInfos.sort((a, b) => {
+        return b.name.localeCompare(a.name);
+      });
+    } else {
+      deviceInfos.sort((a, b) => {
+        return a.name.localeCompare(b.name);
+      });
+    }
```

Comment:
Can we shorten this?  Or at least we can have a general function for sorting that can be used anywhere else.